benchmark/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@loopback/benchmark",
-	"version": "1.1.12",
+	"version": "1.1.13",
```

Comment:
Please revert changes in all package-lock files.

I believe we are no longer including LB4 packages in our lock files, see https://github.com/strongloop/loopback-next/commit/1d658fb6f9e0617e090adcd6265851c01ca8d375